.gitignore

```diff
@@ -1,4 +1,4 @@
-# gitignore file for WordPress Core
+/bin/bash: line 1: /: Is a directory
```

Comment:
?

Reply:
Hmm, interesting, that must have been some sort of copy/paste error. I'll get that fixed.